src/components/Modals/AgreeDisagreeModal.js

```diff
@@ -11,17 +11,19 @@ function AgreeDisagreeModal({
   agreeButtonType,
   children,
   style,
+  buttonLeft,
+  buttonRight,
 }) {
   return (
     <div className={classNames(styles.agreeDisagree, style)}>
       <h3>{text}</h3>
       {children || <span>This operation costs 100 LLD.</span>}
       <div className={styles.buttons}>
         <Button type="button" onClick={onDisagree} medium>
-          Cancel
+          {buttonLeft || 'Cancel'}
```

Comment:
This seems unnecessary? Was it needed or just a preference?

Reply:
I used it in popup, when leaving page without saving. Made this popup more usable.

```
 <AgreeDisagreeModal
          text="You have unsaved changes. What would you like to do with them?"
          buttonLeft="Discard"
          buttonRight="Update"
          style={stylesModal.getCitizenshipModal}
          onDisagree={handleDiscardChanges}
          onAgree={() => updateNominations(selectedValidatorsAsTargets)}
        >
```